### DISCUSSION TITLE
搜索功能&CSS相关问题

### DISCUSSION THREAD

**Discussion by ubreakifix0925** [YunYouJun/hexo-theme-starry#138] [Q&A 疑难杂症] [answered]
主题版本号：1.6.1
云游君，你好！
我在页面上设置了本地搜索的功能，在本地页面运行良好，如图

![](https://ss.im5i.com/2021/07/08/gNpyw.png)

![](https://ss.im5i.com/2021/07/08/gN7RZ.png)

我在yun.yml上的配置：

![](https://ss.im5i.com/2021/07/08/gNMpJ.png)

可是一但部署到GitHub上时，搜索就到页面左下底端了，也没有样式，如图：

![](https://ss.im5i.com/2021/07/08/gNkP1.png)

还有一个问题，每次g或d页面的时候，仓库的hexo-theme-yun.css样式文件都不更新，导致无法显示后面添加的背景，搜索功能无法显示也是这个造成的吧？

yun真是一款相当美观的hexo主题，对于搭建我的博客起到了巨大的作用，感谢云游大大设计出这样好的作品，也希望云游君可以帮我解答疑难，万分感谢！

一个初识网页的学生党

**Comment by YunYouJun** [comment, marked as answer]
主题相关的讨论和使用问题将会被移动至 Discussions。

首先请确保您已开启本地搜索功能，此外请检查是否使用了 CDN，或是否因为浏览器缓存、是否成功生成静态文件并部署等原因造成。

如果仍然无法解决，请提供您的可复现仓库，以及预览地址，以便定位问题。

**Reply by ubreakifix0925** [reply]
就是我每次使用有关于样式功能时都无法更新，样式文件不能自动更改，怎么解决啊

**Reply by YunYouJun** [reply]
请提供你的源代码仓库与预览地址。

https://yun.yunyoujun.cn/guide/faq.html#%E8%87%AA%E6%9F%A5%E9%80%9A%E7%94%A8%E9%97%AE%E9%A2%98%E8%A7%A3%E5%86%B3%E6%96%B9%E6%A1%88

**Reply by ubreakifix0925** [reply]
哦，现在问题解决了，打扰了，谢谢！

**Reply by ubreakifix0925** [reply]
就是把本地的deploy和public删了一遍再g一次，现在好了
